notebooks/week3/03.model_serving_feature_lookup.py

```
@@ -0,0 +1,160 @@
+        ]
+    ),
+)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Call the endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff (0.8.2)

`dbutils` may be undefined, or defined from star imports

(F405)